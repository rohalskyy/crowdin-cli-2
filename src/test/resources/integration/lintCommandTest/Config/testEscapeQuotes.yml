"project_id" : "1"
"api_token" : "a"
"base_path" : ""
"base_url": ""

"preserve_hierarchy": true

files: [
    {
        "source" : "/*.xml",
        "translation" : "/%two_letters_code%/%original_file_name%",
        "escape_quotes": 1
    },
    {
        "source": "/*.xliff",
        "translation": "/%two_letters_code%/%original_file_name%",
        "escape_quotes": 5
    }
]
