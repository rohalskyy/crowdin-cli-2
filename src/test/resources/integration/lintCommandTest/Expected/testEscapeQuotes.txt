Configuration file is invalid. Check the following parameters in your configuration file:
	- Acceptable values for 'escape_quotes are: 0, 1, 2, 3. Default is 3. https://support.crowdin.com/configuration-file-v3/#escape-quotes-options-for-properties-file-format
